diesel/src/macros/mod.rs

```diff
@@ -1055,6 +1055,19 @@ macro_rules! not_none {
     };
 }
 
+/// Gets the value out of an option, or returns an error.
+///
+/// This is used by `FromSql` implementations for Postgres PgValue.
+#[macro_export]
+macro_rules! not_none_pg {
```

Comment:
Hm... I was more envisioning `fn bytes(&self) -> &[u8]` on `PgValue`, and then we'd just do `not_none!(value.bytes())`